cmd/server/workflow/workflow_service.go

```diff
@@ -0,0 +1,167 @@
+package workflow
+
+import (
+	"bufio"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	"github.com/argoproj/argo/pkg/client/clientset/versioned"
+	"github.com/argoproj/argo/workflow/util"
+)
+
+type kubeService struct {
```

Comment:
This is an empty struct. I think we should delete this and inline each method into workflow_service @sarabala1979